src/components/LuxMenuBar.vue

```diff
@@ -630,7 +632,6 @@ export default {
 
   a,
   .lux-submenu-toggle {
-    &:hover,
```

Comment:
On hover in shade theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-black);
}